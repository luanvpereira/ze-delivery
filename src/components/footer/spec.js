import React from 'react';
import { mount } from 'enzyme';

import Layout from './';

describe('<Layout/>', () => {
	it('shoudl render a <Layout/>', () => {
		const component = mount(<Layout />);

		expect(component).toMatchSnapshot();
	});
});
