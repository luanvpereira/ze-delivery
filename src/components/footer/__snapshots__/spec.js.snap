// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout/> shoudl render a <Layout/> 1`] = `
<Footer>
  <footer
    className="footer"
  >
    <Container>
      <div
        className="container"
      >
        <p
          className="footerCopyright"
        >
          © 
          2019
           Zé Delivery - Todos os direitos reservados.
        </p>
        <p
          className="footerLogo"
        >
          Ze Delivery
        </p>
        <ul
          className="footerList"
        >
          <li>
            <a
              className="footerListItem"
              href="https://zedelivery.zendesk.com/hc/pt-br"
              rel="noopener noreferrer"
              target="_blank"
            >
              Dúvidas?
            </a>
          </li>
          <li>
            <a
              className="footerListItem"
              href="/terms"
            >
              Termos de Condições e Política de 
              <br />
               Privacidade
            </a>
          </li>
          <li>
            <a
              className="footerListItem"
              href="/partner"
            >
              Seja nosso parceiro
            </a>
          </li>
        </ul>
         
      </div>
    </Container>
  </footer>
</Footer>
`;
